### DISCUSSION TITLE
TheWicklowWolf suite (multiple requests)

### DISCUSSION THREAD

**Discussion by TheWicklowWolf** [selfhst/icons#122] [New Icon Request]
1. **BookBounty** - Retrieve missing Readarr books from Library Genesis  
   - Project URL: [BookBounty](https://github.com/TheWicklowWolf/BookBounty)  
   - Logo location: [https://github.com/TheWicklowWolf/BookBounty/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/BookBounty/blob/main/src/static/logo.png)

2. **SpotTube** - Download Spotify playlists, artists, albums or tracks via yt-dlp  
   - Project URL: [SpotTube](https://github.com/TheWicklowWolf/SpotTube)  
   - Logo location: [https://github.com/TheWicklowWolf/SpotTube/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/SpotTube/blob/main/src/static/logo.png)

3. **Syncify** - Download Spotify or YouTube playlists on a schedule via yt-dlp  
   - Project URL: [Syncify](https://github.com/TheWicklowWolf/Syncify)  
   - Logo location: [https://github.com/TheWicklowWolf/Syncify/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/Syncify/blob/main/src/static/logo.png)

4. **LidaTube** - Find and download missing Lidarr albums via yt-dlp  
   - Project URL: [LidaTube](https://github.com/TheWicklowWolf/LidaTube)  
   - Logo location: [https://github.com/TheWicklowWolf/LidaTube/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/LidaTube/blob/main/src/static/logo.png)

5. **ChannelTube** - Download Video or Audio from YouTube channels on a schedule via yt-dlp  
   - Project URL: [ChannelTube](https://github.com/TheWicklowWolf/ChannelTube)  
   - Logo location: [https://github.com/TheWicklowWolf/ChannelTube/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/ChannelTube/blob/main/src/static/logo.png)

6. **RadaRec** - Movie discovery tool that provides recommendations based on selected Radarr movies, using TMDB  
   - Project URL: [RadaRec](https://github.com/TheWicklowWolf/RadaRec)  
   - Logo location: [https://github.com/TheWicklowWolf/RadaRec/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/RadaRec/blob/main/src/static/logo.png)

7. **SonaShow** - Show discovery tool that provides recommendations based on selected Sonarr shows, using TMDB  
   - Project URL: [SonaShow](https://github.com/TheWicklowWolf/SonaShow)  
   - Logo location: [https://github.com/TheWicklowWolf/SonaShow/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/SonaShow/blob/main/src/static/logo.png)

8. **PlaylistDir** - Convert a folder of music into a .m3u playlist  
   - Project URL: [PlaylistDir](https://github.com/TheWicklowWolf/PlaylistDir)  
   - Logo location: [https://github.com/TheWicklowWolf/PlaylistDir/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/PlaylistDir/blob/main/src/static/logo.png)

9. **eBookBuddy** - Book discovery tool that provides recommendations based on selected Readarr books, using Goodreads  
   - Project URL: [eBookBuddy](https://github.com/TheWicklowWolf/eBookBuddy)  
   - Logo location: [https://github.com/TheWicklowWolf/eBookBuddy/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/eBookBuddy/blob/main/src/static/logo.png)

10. **Lidify** - Music discovery tool that provides recommendations based on selected Lidarr artists, using Spotify or LastFM  
   - Project URL: [Lidify](https://github.com/TheWicklowWolf/Lidify)  
   - Logo location: [https://github.com/TheWicklowWolf/Lidify/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/Lidify/blob/main/src/static/logo.png)

11. **Huntorr** - Find torrent links and send them to qBitTorrent  
   - Project URL: [Huntorr](https://github.com/TheWicklowWolf/Huntorr)  
   - Logo location: [https://github.com/TheWicklowWolf/Huntorr/blob/main/src/static/logo.png](https://github.com/TheWicklowWolf/Huntorr/blob/main/src/static/logo.png)

**Comment by shollyethan** [comment]
Hi, I love your platform of apps! Any chance you have an SVG format of the logos available to share?

**Reply by TheWicklowWolf** [reply]
No, unfortunately I don't have any SVG logos

**Reply by shollyethan** [reply]
No worries -they PNG versions have been added. Thanks!
